### ISSUE TITLE
[Enhancement] Force v6/v4 for ping, trace etc

### ISSUE THREAD

**Issue by FredNeuberger** [respawner/looking-glass#34]
Hey there,

Wouldn't it be nice to be able to copy paste paste a hostname and force the LG to use a specific IP version?

**Comment by respawner** [comment]
It could be an interesting feature.

I believe that it could be a configuration option like `supported_ip_version`. But should it be global or per router? I can see a use case for per router but it would involve some Javascript to hide/show a new field in the form.

What do you guys think?

**Comment by pautiina** [comment]
**supported_ip_version** -  If this option is entered, then for per router

about copying the host address and inserting it is a good idea.

**Comment by martijn-schmidt** [comment]
I actually discussed such a feature as a potential "long term wish list" item with my colleagues as well, but rather than forcing it to one IP version we figured we'd simply have the looking glass perform ping/traceroute for both address-families where A and AAAA records are available - much like the behaviour for the AS and regex lookup commands.

**Comment by respawner** [comment]
That's actually not a bad idea. The current behavior is to use IPv6 if it is available (we are in 2018 for god sake). But using both IPv6 and IPv4 could be much simpler than implementing a `supported_ip_version` feature.

I do not exclude implementing a feature to force the IP version to use per router. I have been asked for this kind of features several times. But I guess we can do both of ideas. One should just be much faster and easier to implement.

**Comment by FredNeuberger** [comment]
Yes, if both of them are getting processed, I guess thats almost enough. Can't think of any case where you'd want to enter a hostname and really force the address family and not get the other one as well.

**Comment by respawner** [comment]
Let's do that this way then 👍
